include/aspect/material_model/visco_plastic.h

```diff
@@ -254,12 +254,20 @@ namespace aspect
          * A function that returns whether the material is plastically yielding at
          * the given pressure, temperature, composition, and strain rate.
          */
+        DEAL_II_DEPRECATED
         bool
         is_yielding ( const double &pressure,
                       const double &temperature,
                       const std::vector<double> &composition,
                       const SymmetricTensor<2,dim> &strain_rate) const;
 
+        /**
+        * A function that returns whether the material is plastically yielding at
+        * the given pressure, temperature, composition, and strain rate.
```

Comment:
Maybe update the comment?

Reply:
Yes, it would be good to indicate in the comments how the two versions of `is_yielding` differ and where they will be used.